terraform/ecs/variables.tf

```diff
@@ -28,3 +28,7 @@ variable "mock_endpoint" {
 variable "ecs_taskdef_directory" {
   default = "defaults"
 }
+
+variable "ecs_taskdef_network_mode" {
+  default = "vpc"
```

Comment:
Is vpc a valid default value? should it be `awsvpc`?

Reply:
Yea, you are right it should be awsvpc. Thanks!